packages/components/src/components/carousel/carousel-styles.ts

```diff
@@ -1,76 +1,144 @@
+    },
+    [mediaQueryXXL]: {
+      padding: `0 calc(${gridSafeZoneXXL} + ${gridGap} + ${gridColumn1FrXXL})`,
+    },
+  },
+  extended: {
+    padding: `0 ${gridSafeZoneBase}`,
+    [mediaQueryXXL]: {
+      padding: `0 ${gridSafeZoneXXL}`,
+    },
+  },
+};
 
 export const getComponentCss = (
-  wrapContent: boolean,
+  width: CarouselWidth,
   disablePagination: BreakpointCustomizable<boolean>,
+  splideSpeed: number,
+  alignHeader: CarouselAlignHeader,
   theme: Theme
 ): string => {
-  const { primaryColor, disabledColor } = getThemedColors(theme);
+  const { primaryColor, contrastMediumColor } = getThemedColors(theme);
+  const bulletTransitionDuration = `${splideSpeed}ms`;
+  const isHeaderAlignCenter = alignHeader === 'center';
 
   return getCss({
     '@global': {
       ':host': addImportantToEachRule({
-        display: 'grid',
+        display: 'flex',
+        gap: spacingFluidMedium,
+        flexDirection: 'column',
         maxWidth: gridWidthMax,
-        marginLeft: 'auto',
-        marginRight: 'auto',
-        gap: pxToRemWithUnit(24),
-        gridAutoFlow: 'row',
-        [mediaQueryXl]: {
-          gap: pxToRemWithUnit(32),
-        },
+        marginLeft: 'auto', // relevant for viewport width > 2560px
+        marginRight: 'auto', // relevant for viewport width > 2560px
         ...hostHiddenStyles,
       }),
-      'h2,::slotted([slot=heading])': addImportantToEachRule({
-        ...headingMediumStyle,
+      [selectorHeading]: addImportantToEachRule({
+        ...headingXLargeStyle,
+        maxWidth: '56.25rem',
         margin: 0,
-        color: primaryColor,
-        maxWidth: pxToRemWithUnit(900),
       }),
-      'p,::slotted([slot=description])': addImportantToEachRule({
+      [selectorDescription]: addImportantToEachRule({
         ...textSmallStyle,
-        margin: 0,
+        maxWidth: '34.375rem',
+        margin: `${spacingFluidXSmall} 0 0`,
+      }),
+      [`${selectorHeading},${selectorDescription}`]: addImportantToEachRule({
         color: primaryColor,
-        maxWidth: pxToRemWithUnit(550),
-        gridColumn: 1, // to force it into 2nd line
+        [mediaQueryS]: isHeaderAlignCenter
+          ? {
+              gridColumnStart: 2,
+              textAlign: 'center', // relevant when text becomes multiline
+            }
+          : {
+              gridColumn: '1 / 3',
+            },
       }),
     },
+    header: {
+      display: 'grid',
+      ...mergeDeep(spacingMap[width], {
+        [mediaQueryS]: {
+          fontFamily, // relevant for button group width calculation, which is based on ex unit
+          columnGap: spacingStaticMedium,
+          gridTemplateColumns: `${buttonGroupWidth} minmax(0px, 1fr) ${buttonGroupWidth}`,
+          ...(isHeaderAlignCenter && {
+            justifyItems: 'center', // relevant when max-width of heading or description is reached
+          }),
+        },
+      }),
+    },
+    nav: {
```

Comment:
maybe we want to improve alignment, when header is used with or without description